client/ifup.c

```diff
@@ -452,27 +452,35 @@ ni_nanny_fsm_monitor_free(ni_nanny_fsm_monitor_t *monitor)
 static int
 ni_do_ifup_nanny(int argc, char **argv)
 {
-	enum  { OPT_HELP, OPT_IFCONFIG, OPT_CONTROL_MODE, OPT_STAGE, OPT_TIMEOUT,
-		OPT_SKIP_ACTIVE, OPT_SKIP_ORIGIN, OPT_PERSISTENT, OPT_TRANSIENT,
+	enum {
+		OPT_IFCONFIG		= 'c',
+		OPT_HELP		= 'h',
+		OPT_SKIP_ACTIVE		= 'i',
```

Comment:
Are you using some random generator to assign the short options?

We should rather test what all the "tweaks" like --skip-active/origin and so on are doing. Do they need short-opts at all?

Reply:
I've removed some shortcuts for less-common options to minimize the number of conflicts / inconsistencies